spec.bs

```diff
@@ -3178,10 +3173,8 @@ The <dfn for=Navigator method>getInterestGroupAdAuctionData(|config|)</dfn> meth
   1. Let |requestId| be the [=string representation=] of a [=version 4 UUID=].
   1. Set |result|'s {{AdAuctionData/requestId}} field to |requestId|.
   1. Let |context| be the result of serializing |igMap| using |config| into |result|'s

```

Comment:
does this step mean "1. let |context| be .... 1. Set |result|'s {...} to |context|"?

Reply:
No, it's more like
```
1. Let (|requestBlob|, |context|) be the result of serializing |igMap| using |config|.
2. Set |result|'s {{AdAuctionData/request}} to |requestBlob|.
```

Reply:
let's change it to this then. It's much clearer.

Reply:
Done.